templates/default/solr_context.xml.erb

```diff
@@ -1,8 +0,0 @@
-<!--
-  Context File for Apache Solr Web App, Generated by Chef
-  Local modifications will be overwritten by Chef
--->
-<Context docBase="<%= node['drupal_solr']['war_dir'] %>/solr.war" debug="0" crossContext="true">
-  <Environment name="solr/home" type="java.lang.String" value="<%= node['drupal_solr']['home_dir']%>" override="true" />
```

Comment:
@chrisolof this specifically is what I was concerned about. node['drupal_solr']['home_dir'] could be different on local/dev/stage/prod